geode-core/src/main/java/org/apache/geode/distributed/ConfigurationProperties.java

```diff
@@ -1929,6 +1929,19 @@
    * Geode 1.0
    */
   String DISTRIBUTED_TRANSACTIONS = "distributed-transactions";
+
+  /**
+   * The static String definition of the <i>"ssl-use-default-sslcontext"</i> property
+   *
+   * <U>Description</U>If true allows users to use the default SSLContext. When using
+   * default SSLContext, users do not need to configure other ssl properties except
+   * ssl-enabled-components and ssl-require-authentication.
+   * </p>
+   * <U>Default</U>: "false"
+   * </p>
+   * <U>Since</U>: Geode 1.8
+   */
+  String SSL_USE_DEFAULT_SSLCONTEXT = "ssl-use-default-sslcontext";
```

Comment:
I think the javadoc for ssl-use-default-sslcontext needs a bit of work. Users *do* need to either configure the default context, or they need to use the public CA model of trust and get their server certificates signed by a CA that the JDK trusts if they set this property.

Reply:
reworked on the java docs.